<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeItemDialog</class>
 <widget class="QDialog" name="ShapeItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>线颜色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="penColor">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>线样式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="penStyle">
         <property name="currentText">
          <string>No Line</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>No Line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Solid Line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dash Line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dot Line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dash Dot Line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dash Dot Dot Line</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>填充颜色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="brushColor">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>填充样式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="brushStyle">
         <item>
          <property name="text">
           <string>No Brush</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SolidPattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dense1Pattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dense2Pattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dense3Pattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dense4Pattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dense5Pattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dense6Pattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dense7Pattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>HorPattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>VerPattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CrossPattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BDiagPattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FDiagPattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DiagCrossPattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LinearGradientPattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ConicalGradientPattern</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RadialGradientPattern</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShapeItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShapeItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
